src/app/services/store.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { Observable } from 'rxjs';
+
+import { Store } from '../models/store';
+import { enviroment } from 'src/environments/environment.dev';
```

Comment:
change import path to relative, the same as the above, starting with ../ do not absolute path